app/src/main/java/com/iron/espresso/presentation/home/study/list/StudyListActivity.kt

```diff
@@ -0,0 +1,93 @@
+package com.iron.espresso.presentation.home.study.list
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.MenuItem
+import androidx.fragment.app.Fragment
+import com.google.android.material.tabs.TabLayout
+import com.iron.espresso.R
+import com.iron.espresso.base.BaseActivity
+import com.iron.espresso.databinding.ActivityStudyListBinding
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class StudyListActivity :
+    BaseActivity<ActivityStudyListBinding>(R.layout.activity_study_list) {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setToolbarTitle(getString(R.string.study))
+        setNavigationIcon(R.drawable.ic_back_24)
+
+        val fragmentManager = supportFragmentManager
+
+        fragmentManager.beginTransaction()
+            .replace(
+                R.id.study_list_container,
+                NewListFragment.newInstance(),
+                resources.getString(R.string.recency)
+            ).commit()
+
+        val studyTabList = resources.getStringArray(R.array.study_tab)
+        studyTabList.forEach { title ->
+            binding.topTab.addTab(binding.topTab.newTab().setText(title))
+        }
+
+        binding.topTab.addOnTabSelectedListener(object : TabLayout.OnTabSelectedListener {
+            override fun onTabSelected(tab: TabLayout.Tab?) {
+
+                val addFragment = getFragment(tab?.position ?: 0)
```

Comment:
```suggestion
                tab?.position ?: return
                val addFragment = getFragment(tab?.position)
```
요런식이 좋을 듯? 위에같은 경우는 null이면 0으로 진행하니깐

Reply:
수정하였습니다